packages/gestalt/src/InternalLink.js

```diff
@@ -148,6 +151,17 @@ const InternalLinkWithForwardRef: AbstractComponent<Props, HTMLAnchorElement> =
       : {},
   );
 
+  // useOnNavigation is only accessible with Gestalt Provider
+  // and when onNavigation prop is passed to it
+  const { onNavigation } = useOnNavigation() || {};
+
+  const onNavigationClick =
+    onNavigation &&
+    onNavigation({
```

Comment:
optional chaining works for function calls as well

Reply:
DONE